ruby/vampires.rb

```diff
@@ -0,0 +1,71 @@
+#Clint Thacker
+#CHI Nighhawks
+#Vampire Detection program
+
+puts "Number of employees to be processed:"
+count = gets.chomp.to_i
+
+until count == 0
+  puts "Name:"
+  name = gets.chomp
+  puts "Age:"
+  age = gets.chomp.to_i
+  puts "Birth year:"
+  birth_year = gets.chomp.to_i
+  true_age = 2017 - birth_year
+  puts "Enjoy some delicious garlic bread? (y/n)"
+  garlic_bread = gets.chomp
+  puts "Enroll in our mortal health insurance plan? (y/n)"
+  insurance = gets.chomp
+
+  puts "Please name any allergies (enter 'done' when finished):"
+  while user_input = gets.chomp
```

Comment:
While is most useful when using a comparison to control the loop, assigning a variable here does not make the best use of this structure. You might as well use `loop do` instead.

If you use a conditional instead of an assignment, you can avoid using break later in your code.